azext_edge/edge/providers/edge_api/akri.py

```diff
@@ -9,11 +9,11 @@
 
 
 class AkriResourceKinds(ListableEnum):
-    CONNECTORINSTANCE = "connectorinstance"
+    CONNECTOR = "connector"
     CONNECTORTEMPLATE = "connectortemplate"
     DISCOVERYHANDLER = "discoveryhandler"
 
 
-AKRI_API_V1B1 = EdgeResourceApi(group="akri.microsoft.com", version="v1beta1", moniker="akri")
+AKRI_API_V1 = EdgeResourceApi(group="akri.iotoperations.azure.com", version="v1", moniker="akri")
```

Comment:
For back compat with support bundle (just in case) I think we should make this a new API object
```suggestion
AKRI_API_V1B1 = EdgeResourceApi(group="akri.microsoft.com", version="v1beta1", moniker="akri")
AKRI_API_V1 = EdgeResourceApi(group="akri.iotoperations.azure.com", version="v1", moniker="akri")
```

Reply:
Done.